ext/opentelemetry-ext-aiohttp-client/setup.cfg

```diff
@@ -14,7 +14,7 @@
 #
 [metadata]
 name = opentelemetry-ext-aiohttp-client
-description = OpenTelemetry aiohttp client integration
+description = OpenTelemetry aiohttp client instrumentation
```

Comment:
Did you mean to include these in this PR or a separate one? LGTM either way

Reply:
Yeah will leave it to another PR :)